test/test_bandedlu.jl

```diff
@@ -24,11 +24,7 @@ let
     @test BandedMatrices._promote_to_blas_type(typ, Int16)      == typ
     @test BandedMatrices._promote_to_blas_type(typ, Int8)       == typ
 
-    if VERSION < v"0.5.0-dev+5122"
```

Comment:
I'm not in the middle of bisecting something here right now, but it happens fairly often when people report bugs that involve package code.